RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
+    correctionPhaseNS = cms.double(0.0),
+    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
```

Comment:
From the ORP on Tuesday I understood that there will be a full proper digi2raw -> raw2digi chain and the default for QIE10 inputs will be just the hcalDigis.
This setup looks like a temporary bypass for a case of a missing packer (and no expected actual detector data).
Please clarify the situation.